src/cljs/athens/devcards/styling_with_stylefy.cljs

```diff
@@ -110,6 +110,8 @@
 
 (defcard-doc "
   Apply sub-styles with Stylefy's `(use-sub-style)` function.
+              
+  Don't create empty sub-styles, as this may silently break Stylefy processing.
```

Comment:
worth making a PR to stylefy?

Reply:
Not an area I can reasonably expect to make helpful changes, but I'll think about what to do.